package.json

```diff
@@ -21,19 +21,24 @@
       "require": "./dist/quote.umd.js"
     }
   },
+  "types": "./dist/index.d.ts",
```

Comment:
types also should be declared under "exports", take look here: https://github.com/editor-js/nested-list/pull/73/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R23

Reply:
done👍🏼